python/tidy/servo_tidy/tidy.py

```diff
@@ -295,9 +295,13 @@ def find_reverse_dependencies(dependency, version, content):
 def check_toml(file_name, lines):
     if not file_name.endswith(".toml"):
         raise StopIteration
+    mpl_licensed = False
     for idx, line in enumerate(lines):
         if line.find("*") != -1:
             yield (idx + 1, "found asterisk instead of minimum version number")
+        mpl_licensed |= ('license = "MPL-2.0"' in line)
+    if not mpl_licensed:
+        yield (0, ".toml file should contain MPL-2.0 license.")
```

Comment:
I know it's the case that `servo_tidy` might be used for other Servo independent crates. [Here's an example](https://github.com/servo/webrender_traits/blob/master/.travis.yml#L23). For that particular example though, it's fine since that project is MPLv2. It might be problematic for non MPLv2 examples like rust-url. Should one be worried about that scenario?